Dockerfile

```diff
@@ -1,45 +1,60 @@
 FROM ruby:2.3
 
-RUN apt-get update && \
-    apt-get install --no-install-recommends -y cron && \
-    rm -rf /var/lib/apt/lists/* && \
-    apt-get clean
+RUN SUPERCRONIC_URL=https://github.com/aptible/supercronic/releases/download/v0.1.3/supercronic-linux-amd64 && \
+    SUPERCRONIC_BIN=/usr/local/bin/supercronic && \
+    SUPERCRONIC_SHA1SUM=96960ba3207756bb01e6892c978264e5362e117e && \
+    curl -fsSL -o "${SUPERCRONIC_BIN}" "${SUPERCRONIC_URL}" \
+    && echo "${SUPERCRONIC_SHA1SUM}  ${SUPERCRONIC_BIN}" | sha1sum -c - \
+    && chmod +x "${SUPERCRONIC_BIN}"
 
 ENV RAILS_ENV=production \
+    RAILS_LOG_TO_STDOUT=true \
     RAILS_SERVE_STATIC_FILES=true
 
 WORKDIR /usr/src/app
-COPY . ./
-RUN echo $SOURCE_COMMIT > REVISION
 
-RUN buildDeps='libmagic-dev' && \
+# separately from rest of the code to avoid re-running bundler when Gemfile doesn't change
+# plugins is copied completely, because COPY can't pick wildcard directories
+COPY plugins plugins/
+COPY Gemfile Gemfile.lock ./
+
+RUN BUILD_DEPS='libmagic-dev' && \
     apt-get update && \
-    apt-get install --no-install-recommends -y $buildDeps && \
-    rm -rf /var/lib/apt/lists/* && \
-    bundle install --deployment --without development test && \
-    apt-get purge -y --auto-remove $buildDeps && \
-    echo "Foodsoft::Application.config.secret_token = ENV['SECRET_KEY_BASE']" > config/initializers/secret_token.rb && \
-    mkdir -p log && \
-    ln -sfn /dev/stdout log/production.log && \
-    bundle exec whenever --update-crontab
-
-# Add a temporary mysql-server for assets precompilation
+    apt-get install --no-install-recommends -y $BUILD_DEPS && \
+    echo 'gem: --no-document' >> ~/.gemrc && \
```

Comment:
I agree with avoiding installation of documentation, but is there no better way to configure that? this looks very ugly to me

Reply:
Found this in other `Dockerfile`s :/ Will look a bit.

Reply:
Looked a bit in the gem/bundle docs, did not find a different way. See [gem environment](http://guides.rubygems.org/command-reference/#gem-environment) and [bundle install](https://bundler.io/v1.15/man/bundle-install.1.html). The former has no setting options, the latter doesn't mention changing gem environment settings.
I think it isn't too bad to set a configuration option like this? Will remove the file afterwards.